tests/framework/extensions/nodes/nodes.go

```diff
@@ -0,0 +1,63 @@
+package nodes
+
+import (
+	"context"
+
+	"github.com/rancher/rancher/pkg/api/scheme"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var NodeGroupVersionResource = schema.GroupVersionResource{
```

Comment:
Can you write comment explaining this Tim. You can look at the other extensions to get the "template" of what I mean